### ISSUE TITLE
Who uses wet-boew-drupal?

### ISSUE THREAD

**Issue by Pacoup** [wet-boew/wet-boew-drupal#449]
We've been told many departments are now using Drupal, but not the wet-boew distribution specifically, as far as we know. Does anyone know who uses wet-boew-drupal and how it's been going for them so far?

**Comment by joejoseph00** [comment]
City of Ottawa is in the process of putting it into production use and will be the first major implementers of recent versions of wet-boew-drupal.
OTC-CTA will have it implemented by late spring early summer 2013 and we are currently putting it through it's paces as we develop PD and will be doing more preparatory work on it
HC-SC is working on it
citt-tcce will be onboard soon
RCMP-GRC are in
Stats Canada is driving this
Treasury Board is working with us on this
SSC is recommending it and I presume will also be using it.

not sure who's gone live with this distro in production
However our OTC-CTA internet site is using drupal 6 and our intranet site is on drupal 7 (similar to the wet-boew drupal distro in many ways) , 
CRTC is in production with D7 for consultations (correct me if I'm wrong)
NATCAN is using drupal
the PCO is on drupal 7 and we (OTC-CTA) and they are using a version of our customized wysiwyg editor for D7
BuySell 
University of Ottawa is heavily involved, contributing huge amounts of quality code in three important modules that are in the roadmap for the wetboew drupal
Basically it's a collaborative effort, which offers vertical and lateral synergies.

**Comment by StephenOTT** [comment]
@Pacoup if you want to talk about what we are doing (city of ottawa), let me know and we discuss.  We are literally right next door to you.

**Comment by lpgascon** [comment]
Policy Horizons Canada is also using the WetKit for it's next CLF deployment

**Comment by sylus** [comment]
Standards Council of Canada (http://www.scc.ca) is live with an earlier version of WET Drupal as well. ^_^

Closing this issue as feel the question is answered. Thanks everyone :)
